src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,17 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import mate.academy.spring.validator.Email;
+import mate.academy.spring.validator.PasswordMatch;
+
+@PasswordMatch
 public class UserRequestDto {
+    @Email
     private String email;
+    @NotNull
     private String password;
+    @NotNull
```

Comment:
From common mistakes:
Don’t repeat checks for fields: you must ensure email field not to be null in your custom Validator, so there is no need to put @NotNull for that field as well in Dto class. The same goes for the password field.